src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,6 +1,231 @@
 package juja.sqlcmd;
 
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
 public class Application {
+
+    private final static String SERVER = "localhost";
+    private final static String PORT = "5432";
+    private final static String DB_NAME = "sqlcmd";
+    private final static String USER_NAME = "sqlcmd";
+    private final static String PASSWORD = "sqlcmd";
+    private Connection connection;
+
+
     public static void main(String[] args) {
+        try {
+            new Application().simpleSQL();
+        } catch (ClassNotFoundException | SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void simpleSQL() throws ClassNotFoundException, SQLException {
+        String[] users = {"user1|password1", "user2|password2", "user3|password3"};
+        connection = getConnection();
+        deleteTable("user");
+        System.out.println(getTablesName());
+        createTable("user");
+        System.out.println(getTablesName());
+        addUsers(users);
+        showTableData("user");
+        changeUserName("user2", "userсhange1");
+        showTableData("user");
+        deleteUser("user3");
+        showTableData("user");
+        connection.close();
+    }
+
+    /**
+     * Method deletes the user with name "userName" from the table "user".
+     *
+     * @param userName the username you want to delete.
+     * @throws SQLException if a database access error occurs
+     */
+    private void deleteUser(String userName) throws SQLException {
+        if (userName != null) {
+            String query = String.format("DELETE FROM \"user\" WHERE name = '%s'", userName);
+            queryExecute(query);
+        }
+    }
+
+    private void queryExecute(String query) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            statement.executeUpdate(query);
+        }
+    }
+
+    /**
+     * Method changes the username in the "name" field in the "user" table.
+     *
+     * @param oldName old user name. It will be overwritten with the new value.
+     * @param newName new user name value.
+     * @throws SQLException if a database access error occurs
+     */
+    private void changeUserName(String oldName, String newName) throws SQLException {
+        String query = String.format("UPDATE \"user\" SET name = '%s' WHERE name = '%s'", newName, oldName);
+        queryExecute(query);
+    }
+
+    /**
+     * The method prints the table data to the console in the format Field1 | Field2 | ... | FieldN
+     *
+     * @param tableName the name of the table whose data should be output to the console.
+     * @throws SQLException if a database access error occurs
+     */
+    private void showTableData(String tableName) throws SQLException {
```

Comment:
I think, you can move check null into getTableData method and remove this method

Reply:
This case like in line number 120